tests/integration/test_schema.py

```diff
@@ -1237,7 +1379,11 @@ async def test_schema(registry_async_client, trail):
                 "name": "first_name",
                 "type": "string",
             },
-        ]
+            {
+                "name": f"{unique}",
```

Comment:
```suggestion
                "name": unique,
```
`unique` should already be a string. This also applies to the usages below

Reply:
Yes, fixing.